README.md

```diff
@@ -37,6 +37,13 @@ git clone git@github.com:SparkPost/wordpress-sparkpost.git ~/src/wordpress/wp-co
 
 * Activate the plugin from admin panel
 
+## Running Tests
+* Follow the steps in [wp-cli](http://wp-cli.org/docs/plugin-unit-tests/)
```

Comment:
I think it would be a lot clearer if we had the lines needed to run the tests here instead of referencing an outside page that covers a lot more then what we need. (We only need step 3)